src/ItemVirtualMachine.php

```diff
@@ -203,45 +201,48 @@ public static function showForVirtualMachine(CommonDBTM $asset)
             );
 
             if (!empty($hosts)) {
-                echo "<table class='tab_cadre_fixehov'>";
-                echo  "<tr class='noHover'><th colspan='2' >" . __('List of hosts') . "</th></tr>";
-
-                $header = "<tr><th>" . __('Name') . "</th>";
-                $header .= "<th>" . Entity::getTypeName(1) . "</th>";
-                $header .= "</tr>";
-                echo $header;
-
+                TemplateRenderer::getInstance()->display('components/form/item_virtualmachine.html.twig', [
+                    'has_host' => true,
+                ]);
```

Comment:
This is not mandatory to use the template rendering machine to render a single line. Also, your template corresponds to 2 distincts cases that have no parameters in common. This kind of cases should be handled by 2 distinc template files.

```suggestion
                echo '<h3 class="text-start">' . __s('List of hosts') . '</h3>';
```